### PR TITLE
fix Deprecated packages and swarm peers convert error

### PR DESCRIPTION
**Pull request by godcong**
1. changed Deprecated packages to new packages
2. fix swarm peers Latency convert error
2. fixswarm peers address parse error

### CODE REVIEW

File: response.go [hunk 1]
```diff
@@ -69,8 +70,12 @@ func (r *Response) decode(dec interface{}) error {
 	if r.Error != nil {
 		return r.Error
 	}
-
-	err := json.NewDecoder(r.Output).Decode(dec)
+	b, e := ioutil.ReadAll(r.Output)
+	if e != nil {
+		return e
+	}
+	fmt.Println(string(b))
```
Comment (Stebalien) [inline response.go]: Probably didn't mean to check this patch in.

File: swarm.go [hunk 2]
```diff
@@ -98,7 +98,8 @@ func (api *SwarmAPI) Peers(ctx context.Context) ([]iface.ConnectionInfo, error)
 
 		out.addr, err = multiaddr.NewMultiaddr(conn.Addr)
 		if err != nil {
-			return nil, err
+			//ignore error for get some address like this:ip4/94.130.135.167/tcp/4002/ws/ipfs/QmUEMvxS2e7iDrereVYc5SWPauXPyNwxcy9BXZrC1QTcHE
```
Comment (Stebalien) [inline swarm.go]: We probably don't want to _ignore_ this peer. Let's just not set the addr and document this. Relates to https://github.com/multiformats/multiaddr/issues/70.

Comment (godcong) [reply swarm.go]: yes, This is better.

File: response.go (hunk 1 above)
Comment (godcong) [reply response.go]: i reverted it.
i'm sorry to patched it. 
that's only my debug source.

**Pull request merged**